app/javascript/application.js

```diff
@@ -3,7 +3,6 @@
 // includes should be specified at the end of the file, not in this section. This helps avoid merge conflicts in the
 // future should the framework defaults change.
 
-import Rails from "@rails/ujs"
```

Comment:
@yshmarov I think that removing this line would cause the `Rails.start()` call below (on line 12/11) to fail. Do you need to also remove that?

Reply:
If we are removing `rails/ujs` from `package.json`, we won't have a place to "import Rails" from

I can have a look at how/whether `Rails.start()` needs to be updated